### PR TITLE
chore: udpate jest

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
No.

### Motivation / Use-Case
Upgrade jest + typescript.

### Breaking Changes
No.

### Additional Info
No

### CODE REVIEW

File: lib/Server.js [hunk 1]
```diff
@@ -38,18 +38,19 @@ const schema = require("./options.json");
...
 /** @typedef {import("http").IncomingMessage} IncomingMessage */
+/** @typedef {import("webpack-dev-middleware").ServerResponse} ServerResponse */
 /** @typedef {import("open").Options} OpenOptions */
 
 /** @typedef {import("https").ServerOptions & { spdy?: { plain?: boolean | undefined, ssl?: boolean | undefined, 'x-forwarded-for'?: string | undefined, protocol?: string | undefined, protocols?: string[] | undefined }}} ServerOptions */
 
 /**
  * @template Request, Response
- * @typedef {import("webpack-dev-middleware").Options<Request, Response>} DevMiddlewareOptions
+ * @typedef {import("webpack-dev-middleware").Options<IncomingMessage, ServerResponse>} DevMiddlewareOptions
```
Comment (alexander-akait) [inline lib/Server.js:48]: hm, why it was changed?

Comment (snitin315) [reply lib/Server.js:48]: Typescriptv5  not happy with `<Request, Response>` anymore

**Review by alexander-akait — COMMENTED** [review]
hm, I am afraid it can break types for other developers, we need to investigate it

**Comment by snitin315** [comment]
Hmm, could be a bug in typescript too. I'll check

**Review by alexander-akait — APPROVED** [review]

**Comment by snitin315** [comment]
Let's upgrade ts separately

**Pull request merged**